tests/publiccloud/transfer_repos.pm

```diff
@@ -57,7 +57,17 @@ sub run {
         # Mitigate occasional CSP network problems (especially one CSP is prone to those issues!)
         # Delay of 2 minutes between the tries to give their network some time to recover after a failure
         # For rsync the ~/repos/./ means that the --relative will take efect after.
-        script_retry("rsync --timeout=$timeout -uahRd -e ssh --files-from /tmp/transfer_repos.txt ~/repos/./ '$remote:/tmp/repos/'", timeout => $timeout + 10, retry => 3, delay => 120);
+        # * The --relative (-R) option is implied when --files-from is specified.
```

Comment:
Why point this out if it's not being used?

Reply:
This was necessary before we added `--files-from` which implies it.

Reply:
ok